### DISCUSSION TITLE
How To create table through flutter code In Supabase?

### DISCUSSION THREAD

**Discussion by SamarpanCoder2002** [supabase/supabase#2996] [Questions] [answered]
I want to make a chat application, where every user contains a table.. for that when new users register in the app... I have to create the table in supabase for himself/herself manually from flutter code only... not from supabase UI or external Postgres client... Please help me.

**Comment by soedirgo** [comment, marked as answer]
Please stop creating duplicate discussions. It's already answered [here](https://github.com/supabase/supabase/discussions/2981) that creating a table programmatically is not how you do things in RDBMS. This is the place to discuss Supabase-specific stuff, we can't help you build your app.
